lib/src/scss-shared/_nx-focus-border-helpers.scss

```diff
@@ -9,9 +9,8 @@
 // Border size is 1px border and 6px here represents total space in each axis
 // between border and control elements. Values are based on RSC design spec.
 
-// NxRadio and NxCheckbox require the left property for correct positioning of focus border
+// NxRadio and NxCheckbox require the left and top property for correct positioning of focus border
```

Comment:
You say "left and top property" but I only see height and width below

Reply:
> You say "left and top property" but I only see height and width below

The `left` and `top` are currently specified in `NxRadio` and `NxCheckbox`'s respective .scss files
c64660e53c501a27168f5b17d48bf49041aa8cb3